impl/pom.xml

```diff
@@ -104,4 +104,61 @@
             <artifactId>jsonevent-layout</artifactId>
         </dependency>
     </dependencies>
+
+    <build>
```

Comment:
In both ui and impl we have defined this plugin that during install creates docker images and during deploy pushes them to dockerhub.